packages/ember/index.js

```diff
@@ -652,42 +659,6 @@ if (EMBER_GLIMMER_INVOKE_HELPER) {
 }
 Ember._captureRenderTree = captureRenderTree;
 
-if (ENV.EXTEND_PROTOTYPES.String) {
-  String.prototype.htmlSafe = function () {
-    return htmlSafe(this);
-  };
-}
-const deprecateImportFromString = function (
-  name,
-  message = `Importing ${name} from '@ember/string' is deprecated. Please import ${name} from '@ember/template' instead.`
-) {
-  deprecate(message, false, {
-    id: 'ember-string.htmlsafe-ishtmlsafe',
-    for: 'ember-source',
-    since: {
-      enabled: '3.25',
-    },
-    until: '4.0.0',
-    url: 'https://deprecations.emberjs.com/v3.x/#toc_ember-string-htmlsafe-ishtmlsafe',
-  });
-};
-Object.defineProperty(Ember.String, 'htmlSafe', {
-  enumerable: true,
-  configurable: true,
-  get() {
-    deprecateImportFromString('htmlSafe');
-    return htmlSafe;
-  },
-});
-Object.defineProperty(Ember.String, 'isHTMLSafe', {
-  enumerable: true,
-  configurable: true,
-  get() {
-    deprecateImportFromString('isHTMLSafe');
-    return isHTMLSafe;
-  },
-});
```

Comment:
Hmm, don't we still need these properties? Were they already there?

Reply:
I moved them up with the rest of of the `Ember.String` methods: https://github.com/emberjs/ember.js/pull/19396/files#diff-2f99d92a23bae60fccaee9fcfeda8b77ca73723657662cfadfa4d82751045e14R453.
Since we're not calling the deprecation method, I thought they could all be assigned together.

Reply:
D'oh! Sorry about that, I totally missed it in the diff.

> Since we're not calling the deprecation method, I thought they could all be assigned together.

I think that maybe it would be better to keep the infra around for the deprecation, and change the `deprecateImportFromString` to pass `true` to deprecate (with a nice comment explaining that it needs to be changed back to `false` when we have addressed that issue).